### PR TITLE
Add support for 8th language

### PR DESCRIPTION
**Pull request by ErikSchierboom**
This PR adds support for the [8th language](https://8th-dev.com/)

### CODE REVIEW

File: languages.json [hunk 1]
```diff
@@ -1,5 +1,13 @@
 {
   "languages": {
+    "_8th": {
```
Comment (XAMPPRocky) [inline languages.json:3]: I think it would be better to rename this, and move it into the E section
```suggestion
    "Eighth": {
```

Comment (ErikSchierboom) [reply languages.json]: Good idea!

File: languages.json [hunk 2]
```diff
@@ -456,6 +456,14 @@
       "line_comment": [";"],
       "extensions": ["edn"]
     },
+    "Eight": {
```
Comment (XAMPPRocky) [inline languages.json:459]:
```suggestion
    "Eighth": {
```

**Comment by XAMPPRocky** [comment]
Thank you for your PR!

**Pull request merged**